hoot-rnd/src/test/cpp/hoot/rnd/visitors/MatchCandidateCountVisitorRndTest.cpp

```diff
@@ -87,7 +73,7 @@ class MatchCandidateCountVisitorRndTest : public CppUnit::TestFixture
     map->visitRo(uut);
     CPPUNIT_ASSERT_EQUAL((int)36, (int)uut.getStat());
     QMap<QString, long> matchCandidateCountsByMatchCreator =
-      any_cast<QMap<QString, long> >(uut.getData());
+      boost::any_cast<QMap<QString, long> >(uut.getData());
```

Comment:
Having a `using namespace boost` in a file has caused us problems before.  Got rid of that and added `boost::` where it was necessary.